app/helpers/navigation_items.rb

```diff
@@ -67,7 +67,7 @@ def for_provider_primary_nav(current_provider_user, current_controller, performi
     end
 
     def for_provider_account_nav(current_provider_user, current_controller, performing_setup: false)
-      return [] if (active_action?(current_controller, 'new') && !active?(current_controller, 'application_data_export')) || active_action?(current_controller, 'sign_in_by_email')
```

Comment:
Just testing this in the review app, the sign in link now shows on the sign in page

Reply:
interesting. So this must be what 'new' was attempting to do.

Reply:
Could we use the existing `active?` method for the controller name and also check the actions?

We don't need to check that it's the provider interface sessions controller because this method is only used in the provider interface

Reply:
the controller action is `new`, `sign_in` is just the name used for the route, so that won't work. Unless I'm mirsunderstanding?